db/seeds.rb

```diff
@@ -192,16 +197,39 @@ def describe_teacher(teacher)
 
 active_appropriate_bodies = [umber_teaching_school_hub, golden_leaf_teaching_school_hub]
 
+print_seed_info("Adding registration periods")
+
+rp_2021 = RegistrationPeriod.create!(year: 2021, started_on: Date.new(2021, 6, 1), finished_on: Date.new(2022, 5, 31), enabled: false).tap { |ay| describe_registration_period(ay) }
+rp_2022 = RegistrationPeriod.create!(year: 2022, started_on: Date.new(2022, 6, 1), finished_on: Date.new(2023, 5, 31), enabled: false).tap { |ay| describe_registration_period(ay) }
+rp_2023 = RegistrationPeriod.create!(year: 2023, started_on: Date.new(2023, 6, 1), finished_on: Date.new(2024, 5, 31), enabled: true).tap { |ay| describe_registration_period(ay) }
+rp_2024 = RegistrationPeriod.create!(year: 2024, started_on: Date.new(2024, 6, 1), finished_on: Date.new(2025, 5, 31), enabled: true).tap { |ay| describe_registration_period(ay) }
+rp_2025 = RegistrationPeriod.create!(year: 2025, started_on: Date.new(2025, 6, 1), finished_on: Date.new(2026, 5, 31), enabled: true).tap { |ay| describe_registration_period(ay) }
+
 print_seed_info("Adding lead providers")
 
-grove_institute = LeadProvider.create!(name: 'Grove Institute').tap { |dp| describe_lead_provider(dp) }
-LeadProvider.create!(name: 'Evergreen Network').tap { |dp| describe_lead_provider(dp) }
-national_meadows_institute = LeadProvider.create!(name: 'National Meadows Institute').tap { |dp| describe_lead_provider(dp) }
-LeadProvider.create!(name: 'Woodland Education Trust').tap { |dp| describe_lead_provider(dp) }
-LeadProvider.create!(name: 'Teach Orchard').tap { |dp| describe_lead_provider(dp) }
-LeadProvider.create!(name: 'Highland College University').tap { |dp| describe_lead_provider(dp) }
-wildflower_trust = LeadProvider.create!(name: 'Wildflower Trust').tap { |dp| describe_lead_provider(dp) }
-LeadProvider.create!(name: 'Pine Institute').tap { |dp| describe_lead_provider(dp) }
+grove_institute = LeadProvider.create!(name: 'Grove Institute')
```

Comment:
maybe we can use proper lead providers rather than example ones? happy to discuss

Reply:
I was thinking this mostly because eventually we'll create API tokens to test on the review app, for ECF1 we had set and known tokens such as `ambition-token`, `ucl-token` etc for the known lead providers which made testing easy and they were remember by everyone. If we go with random lead providers this would be harder to remember. Lead providers mostly don't change so it'll just make things easier and more standard for testing I think. I think the only real data I'd suggest is provider names, everything else can be fake as it is now

Reply:
I'd agree if we were generating them with Faker, but they're hardcoded. We can definitely use the set and known tokens idea though.

I like having a clear distinction between real and fake and would prefer not to mix up.

Reply:
discussed on slack: we'll use fake names closer to the real ones. I'll open a different pr not part of this one. Thanks